charts/delphai-keda/values.yaml

```diff
@@ -29,6 +29,7 @@ mountPath: /app/data
 fileShares: []
 onePodMaxPerNodePerApp: false
 preferGpu: false
+mlflow: false
```

Comment:
How many apps are using this option? Is it only one?

Reply:
Yes, just the mlflow-server

Reply:
We do have one helmchart shared across all services, it's not very sustainable idea to configure service-level stuff here. We'll end up with giant unsupportable list of secrets / configs weakly linked to their consumers.

It would be better to have helmchart configurable which secrets and how to use, so the app could define what's meaningful for it.

https://atomleap.atlassian.net/browse/DEVOPS-268

Reply:
Same applies to https://github.com/delphai/helm-charts/pull/42